src/controllers/pinController.ts

```diff
@@ -749,11 +793,16 @@ export class PINController extends Controller {
             email: requestBody.email,
             phoneNumber: requestBody.phoneNumber,
         };
-        const errors = await batchUpdatePin(
+
+        const batchUpdatePinResponse = await batchUpdatePin(
             [pinResult[0]],
             emailPhone,
             requestBody.requesterUsername, // TODO: Get info from token
         );
+
+        const errors = batchUpdatePinResponse[0];
+        const regenerateOrCreate = batchUpdatePinResponse[1];
```

Comment:
@hannah-macdonald1 Updated so that regenerate/create is coming from batchUpdatePinResponse function